js/exporter/exceljs/export.js

```diff
@@ -111,17 +111,34 @@ export const Export = {
                         mergedCells[i][j] = true;
                     }
                 }
+
                 return {
                     start: { row: rowIndex, column: cellIndex },
-                    end: { row: rowIndex + (cellMerge.rowspan || 0), column: cellIndex + (cellMerge.colspan || 0) }
+                    end: { row: rowIndex + (cellMerge.rowspan || 0), column: cellIndex + (cellMerge.colspan || 0) },
+                    merged: privateOptions._isRangeMerged(dataProvider, rowIndex, cellIndex, mergeRowFieldValues, mergeColumnFieldValues, cellMerge.rowspan, cellMerge.colspan)
                 };
             }
         }
     },
 
     mergeCells: function(worksheet, topLeftCell, mergeRanges) {
         mergeRanges.forEach((mergeRange) => {
-            worksheet.mergeCells(mergeRange.start.row + topLeftCell.row, mergeRange.start.column + topLeftCell.column, mergeRange.end.row + topLeftCell.row, mergeRange.end.column + topLeftCell.column);
+            const startRow = mergeRange.start.row + topLeftCell.row;
+            const startColumn = mergeRange.start.column + topLeftCell.column;
+            const endRow = mergeRange.end.row + topLeftCell.row;
+            const endColumn = mergeRange.end.column + topLeftCell.column;
+
+            if(mergeRange.merged) {
+                worksheet.mergeCells(startRow, startColumn, endRow, endColumn);
+            } else {
+                const copiedValue = worksheet.getCell(startRow, startColumn).value;
```

Comment:
mergeCells method should 'merge cells', but in new version it doesn't merge cells.
imho, this implementation is too hard to understand and there should be another where mergeCells method does 'merge cells'

Reply:
You are absolutely right. We have revised this solution in future versions. Now this is no longer relevant.